EapImport/src/main/java/de/hpi/unicorn/importer/json/JsonExporter.java

```diff
@@ -88,4 +89,42 @@ public static File generateExportFileWithDependencies(final EapEventType eventTy
         }
         return file;
     }
+
+    public static File generateExportFileWithValues(EapEventType eventType, Map<TypeTreeNode, String> inputValues, int eventCount, int scaleFactor, String timestamp) {
+        // create file
+        if (eventType.isHierarchical()) {
+            return null;
+        }
+        final File file = new File(TempFolderUtil.getFolder() + System.getProperty("file.separator") + eventType.getTypeName() + "export.json");
+        try (FileWriter writer = new FileWriter(file, false)) {
+            // general information
+            writer.append("{\"eventTypeValues\" : ");
+            writer.append("{\"eventType\" : ");
+            writer.append("{\"name\" : \"" + eventType.getTypeName() + "\",");
+            writer.append("\"timeStampName\" : \"" + eventType.getTimestampName() + "\"},");
+            writer.append("\"eventCount\" : " + eventCount + ",");
+            writer.append("\"scaleFactor\" : " + scaleFactor + ",");
+            writer.append("\"timestamp\" : " + timestamp + ",");
+
+            // values
+            writer.append("\"values\" : [");
+            TypeTreeNode[] nodes = {};
+            TypeTreeNode[] attributes = inputValues.keySet().toArray(nodes);
+            for (int i = 0; i < inputValues.size(); i++) {
+                TypeTreeNode node = attributes[i];
+                writer.append("{\"attribute\" : {");
+                writer.append("\"name\" : \"" + node.getName() + "\",");
+                writer.append("\"type\" : \"" + node.getType() + "\"},");
+                writer.append("\"value\" : \"" + inputValues.get(node) + "\"}");
+                if (i != inputValues.size() - 1) {
+                    writer.append(",");
+                }
+            }
+            writer.append("]}}");
+        } catch (final IOException e1) {

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=squid%3AS1166)